docs/translations/zh_CN/usage.md

```diff
@@ -0,0 +1,33 @@
+# 如何使用 syzkaller
```

Comment:
I think it would be great to also add per-page disclaimers (also in Chinese?) like `This is a community-driven translation of the official syzkaller documentation. The most up-to-date version of this page (in English) may be found at docs/usage.md`.